README.md

```diff
@@ -0,0 +1,215 @@
+# @apostrophecms/content-upgrader
+
+A tool to migrate your **content** from Apostrophe 2.x to Apostrophe 3.x. That is, it creates a new database in the A3 format, and copies over the uploaded media. This tool does not upgrade your source code. However notes on some of the relevant code changes you'll need are provided below. See also [Coming from Apostrophe 2.x](https://v3.docs.apostrophecms.org/guide/upgrading.html), especially the Breaking Changes section.
+
+## Stability
+
+**This is an alpha release.** We have not done our own acceptance testing on the output yet. It may be useful to you in A2 migrating sites. Your bug reports and contributions are appreciated.
+
+## Limitations
+
+* As stated this is a content migration tool, not a code migration tool. You will need to make code changes manually to convert an A2 project.
+* Users and groups are not migrated. This is because the user roles of A3 differ in design from the permissions groups of A2 and we wish to avoid creating any security issues. You should create new accounts on the A3 project or arrive at your own migration strategy.
+* A2 has a built-in `apostrophe-images` "slideshow" widget type, while A3 only has a built-in single-image `@apostrophecms/image` widget type. By default `apostrophe-images` will be upgraded to `@apostrophecms/image`, with only the first image present in each. However you can use the `mapWidgetTypes` option, documented below, to override this mapping during the upgrade.
+* Since A3 does not have a built-in cropping feature yet, there is currently no accommodation for it in A3's `@apostrophecms/image` widget type. However an attempt is made to carry over the cropping data in the format which is expected to work in A3 in the near future.
+
+## Installation
+
+This tool installs as a module **inside your existing A2 project.** This is necessary to gain access to information such as the schemas of your existing piece and page and widget types.
+
+```bash
+cd my-existing-a2-project
+npm install @apostrophecms/content-upgrader
+```
+
+## Configuration
+
+Edit `app.js` of your A2 project. Add the module to the `modules` section:
+
+```javascript
+modules: {
+  // ... all of your existing A2 modules here ...
+  '@apostrophecms/content-upgrader': {}
+}
+```
+
+Then create `lib/modules/@apostrophecms/content-upgrader/index.js`. Here you can optionally address any content transformations and set the default locale, which is important *even if you have no immediate plans to localize your site in other languages.*
+
+### Setting the default locale
+
+In `lib/modules/@apostrophecms/content-upgrader/index.js`, be sure to set the default locale name to match your A3 project. Otherwise your site may appear to have no content after the upgrade.
```

Comment:
I don't understand this. if you only have one locale you'll lose content if you don't set `defaultLocale`?

Reply:
My timebox is boxed, this is alpha and I have to hit other tickets today. So I'm opening followup tickets on these:

PRO-2094